compiler/Compiler.lkshw

```diff
@@ -0,0 +1,14 @@
+Version of workspace file format:
```

Comment:
It was added by the IDE I use. Will remove.

---
In reply to: [124936962](https://github.com/Microsoft/bond/pull/517#discussion_r124936962) [](ancestors = 124936962)